engine/engine.go

```diff
@@ -307,11 +322,7 @@ func handleSolveEvents(startOpts *Config, upstreamCh chan *bkclient.SolveStatus)
 		switch u.Scheme {
 		case "":
 			eg.Go(func() error {
-				f, err := os.OpenFile(
-					startOpts.JournalURI,
-					os.O_APPEND|os.O_CREATE|os.O_WRONLY,
-					0644,
-				)
+				f, err := os.Create(startOpts.JournalURI)
```

Comment:
Changing this to truncate instead of appending, per conversation w/ @marcosnils and @aluzzardi